.github/workflows/test-template.yml

```diff
@@ -559,7 +559,7 @@ jobs:
   # CLIENT (types tests only)
   #
   client-types:
-    name: Client (types tests only)
+    name: Client (types)
```

Comment:
```suggestion
    name: Client (legacy types)
```

Reply:
Why is this "legacy types" if we have another test job that is "no types"?